list_of_projects_converter.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+file = open('list_of_projects.txt', 'w')
+
+filecut = os.listdir('./_projects')
+filecut.remove('.placeholder')
+
+for sFile in filecut:
+    content = open('./_projects/'+ sFile, 'r')
+    number = 1
+    filename = ''
+    membersfound = 0
+    members = ''
+    for line in content:
+        if number == 3:
+            filename = line.replace('title: ', '')
+            filename = filename.replace('\n', '')
+        elif line[0] == 'm':
+            membersfound = 1
+        elif membersfound == 1 and line.find('  -')!= -1:
+            members = members + line
+        elif membersfound == 1 and line.find('  -')== -1:
+            membersfound = 0
+        number = number+1
```

Comment:
Do not work with counters.. some projects may shuffle the header a bit (or we might want to change the order of the keywords) and `title` ends up being in line 9 or so. Better search each line (of the header only, the block marked by `---`) for `title:`, `members:`, and so on.